pysolarfocus/const.py

```diff
@@ -1,357 +1,15 @@
 """Constants for pysolarfocs"""
+import logging
+from enum import Enum
 
-INT = 1
-UINT = 2
-
+class Systems(str, Enum):
```

Comment:
I'm wondering, if there is a better place for this class. WDYT?

Reply:
Well idk, we need this enum to initialize the api. I put it here because the port needed to create the modbus client was also there -> both need to be imported to create the client and api. Maybe we should move them to the root of the package then a consuming application could import them with `from pysolarfocus import Systems, PORT`. WDYT?

Reply:
:+1 like it.

Reply:
Alright i will changes this in a commit later.